### PR TITLE
Ritual Inquisitor changes

### PR DESCRIPTION
**Pull request by uilman**
image
Righteous Providence
image
image

Fanaticism
image
Without
image
With
image

Instruments of Virtue
image
image
image
image

Pious Path
image
image
Untethered has life and ES recovery rate. Set life to 15%, ES to 20%
image

### CODE REVIEW

**Comment by uilman** [comment]
Known issue, pious path currently doesn't apply ES regen to minions

**Comment by uilman** [comment]
image

**Comment by uilman** [comment]
image
image

File: Modules/CalcPerform.lua [hunk 1]
```diff
@@ -430,6 +430,12 @@ local function doActorMisc(env, actor)
 			modDB:NewMod("Speed", "INC", effect, "Onslaught")
 			modDB:NewMod("MovementSpeed", "INC", effect, "Onslaught")
 		end
+		if modDB:Flag(nil, "Fanaticism") and actor.mainSkill and actor.mainSkill.skillFlags.selfCast then
+			local effect = m_floor(75 * (1 + modDB:Sum("INC", nil, "BuffEffectOnSelf") / 100))
+			modDB:NewMod("Speed", "MORE", effect, "Fanaticism", ModFlag.Cast)
+			modDB:NewMod("ManaCost", "MORE", -effect, "Fanaticism")
```
Comment (ppoelzl) [inline Modules/CalcPerform.lua:436]: Shouldn't this be "LESS" mana cost instead?

**Review by ppoelzl — APPROVED** [review]
Everything is working correctly now.

**Pull request merged**